html-report-generator/generate_usage_report.py

```diff
@@ -31,7 +31,8 @@ def main():
     # Get directories to query from _config/dirs.txt
     config_file_path = pjoin(working_dir, '_config/') + 'dirs.txt'
     with open(config_file_path, 'r') as file:
-        dir_list = [line.strip() for line in file]
```

Comment:
The way we experts do it is that:

> dir_list = file.read().strip().split('\n')

The last can be left as just `.split()` if you are confident that no line has whitespace in it.

Reply:
Tested and fixed